release-notes/topics/aap-23.adoc

```diff
@@ -17,7 +17,11 @@ Red Hat Ansible Automation Platform simplifies the development and operation of
 * Updated internal FieldAttribute classes to act as Python data descriptors to reduce code complexity and use of metaclasses.
 * Fixed an issue when ansible-runner was not properly removed from hybrid nodes when upgrading to Ansible Automation Platform 2.2.
 
-.Technology preview features
+== Technology preview features
+
+Some features in this release are currently classified as Technology Preview. Technology Preview features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process. Please note that Red Hat does not recommend using technology preview features for production, and Red Hat SLAs do not support technology preview functions.
```

Comment:
Note that Red Hat does not recommend...

IBM style guide says not to use Please.

Reply:
Made this change, thank you.